dbmanager/dbmanager_test.go

```diff
@@ -0,0 +1,73 @@
+package dbmanager
+
+import (
+	"testing"
+)
+
+var PSQL DBManager
+var PSQLerr error
+
+func init() {
+	PSQL, PSQLerr = New("postgres", "myPassword", "iot_dashboard")
+}
+
+func TestNewDBManager(t *testing.T) {
+	//this inherently also tests ConnectToPSQL()
+	if PSQLerr != nil {
+		t.Errorf("Not Able to connect to database")
+	}
+}
+
+func TestVerifyUserExists(t *testing.T) {
+	cases := []struct {
+		email  string
+		exists bool
+	}{
+		{"user@gmail.com", true},
```

Comment:
Similar to my previous comments, I think this assumes that the database contains certain values already. Please either populate the database in the test itself, or mock the DB calls. As this stands, the test is not very useful.

Reply:
Same for the other tests in this file, I won't leave comments on each one of them

Reply:
All tests now use `go-sqlmock` where applicable